java/ql/test/query-tests/security/CWE-489/webview-debugging/WebviewDebuggingEnabled.ql

```diff
@@ -1,15 +1,18 @@
 import java
-import TestUtilities.InlineFlowTest
```

Comment:
You can still use InlineFlowTest - see the couple of conversions I did. I added the new extension points `predicate hasTaintFlow(DataFlow::Node src, DataFlow::Node sink)` and `predicate hasValueFlow(DataFlow::Node src, DataFlow::Node sink)`, which can be used.